client/reader/recent/style.scss

```diff
@@ -176,6 +183,18 @@ body.is-reader-full-post {
 				padding: $recent-feed-spacing;
 			}
 		}
+
+		// This is a temporary fix to disable the pagination page selector because of this issue: Automattic/loop#247
+		// Once that issue is fixed, this code should be removed.
+		.dataviews-pagination__page-select {
```

Comment:
Is it possible to target the `cursor: pointer` on `.dataviews-pagination__page-select > .components-select-control`?

Reply:
Ah, the `cursor` style needed to be added to the container, not the control in this case.